ecs-init/cache/dependencies_mock_test.go

```diff
@@ -1,4 +1,4 @@
-// Copyright 2015-2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// Copyright 2015-2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Are these changed because you updated the `gomock`?

Reply:
Yes, and `make test` automatically updates the mocks in this project.

Reply:
Is that good or bad? wondering what happens if there is error in updating mocks.

Reply:
It means I have to fix them, which is annoying. :smile: 

@adnxn were just talking about solutions to this, but its probably something we wont spend a lot of time on here.